mvcc/backend/read_tx.go

```diff
@@ -28,9 +28,16 @@ import (
 var safeRangeBucket = []byte("key")
 
 type ReadTx interface {
+	// Lock locks txReadBuffer for writing
```

Comment:
we should not leak details of the impl here. remove txReadBuffer mentioning.

Reply:
Thanks! Done.